Fabric.Authorization.API/Configuration/DiscoveryServiceSettings.cs

```diff
@@ -3,7 +3,10 @@
     public class DiscoveryServiceSettings
     {
         public string AccessControlToken { get; set; }
```

Comment:
Does it make sense to rename this to DiscoveryServiceToken?

Reply:
sounds good